src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx

```diff
@@ -13,34 +13,64 @@ After you import OpenTelemetry data into New Relic, you can use a variety of too
 
 ## Explorer: Get the big picture along with the details [#explorer]
 
-The New Relic explorer is a good place to get overview information about your applications, as well as detailed information, such as distributed tracing.
+The New Relic **Explorer** tab is a good place to start gathering information about your services: 
 
-To get started with the explorer:
-
-1. Go to [one.newrelic.com](https://one.newrelic.com) and click **Explorer**.
+1. Go to [one.newrelic.com](https://one.newrelic.com).
 2. In the left sidebar, click **Services - OpenTelemetry**:
 
     ![Screen capture showing how to filter for OpenTelemetry services.](./images/explorer_otel_services.png "Filter for OpenTelemetry to see only those entities.")
 
-3. Click the service you want to know more about.
+3. In the center pane, click the service you want to know more about, or find your service by entering the name in the top filter bar.
 
-If you need help understanding the data, see the explanations which follow.
+If you need help understanding how we display your data, see the left-pane options below.
```

Comment:
would a link enclosing "left-pane options" be more helpful than "below" ? I'm not sure how far down this document I need to skim

Reply:
@barb, the options are immediately following, so I changed it to this: "...see the following descriptions of the left-pane options in the UI:"